### PR TITLE
Refactor page-editor elements to use Composition API

### PR DESCRIPTION
**Pull request by pixeldesu**
# What

Use Composition API for all `page-editor` elements and blocks.

# Why

Refactor to Composition API (per contribution guide)

### CODE REVIEW

File: packages/client/src/pages/page-editor/els/page-editor.el.note.vue [hunk 1]
```diff
@@ -16,52 +16,37 @@
...
-				}
+let id: any = $ref(props.value.note);
+let note: any = $ref(null);
 
-				this.note = await os.api('notes/show', { noteId: this.value.note });
-			},
-			immediate: true
-		},
-	},
+watch(() => id, async () => {
```
Comment (Johann150) [inline packages/client/src/pages/page-editor/els/page-editor.el.note.vue]: `id` is already a `Ref` so I think a getter is unnecessary?
```suggestion
watch(id, async () => {
```

File: packages/client/src/pages/page-editor/els/page-editor.el.note.vue [hunk 2]
```diff
@@ -16,52 +16,37 @@
...
+	if (id && (id.startsWith('http://') || id.startsWith('https://'))) {
+		props.value.note = id.endsWith('/') ? id.substr(0, id.length - 1).split('/').pop() : id.split('/').pop();
```
Comment (Johann150) [inline packages/client/src/pages/page-editor/els/page-editor.el.note.vue]:
```suggestion
		props.value.note = (id.endsWith('/') ? id.slice(0, -1) : id).split('/').pop();
```

File: packages/client/src/pages/page-editor/els/page-editor.el.radio-button.vue [hunk 3]
```diff
@@ -12,41 +12,28 @@
...
+	value: {
+		name: '',
+		title: '',
+		values: []
+	}
+});
+
+let values: string = $ref(props.value.values.join('\n'));
+
+watch(() => values, () => {
```
Comment (Johann150) [inline packages/client/src/pages/page-editor/els/page-editor.el.radio-button.vue]: Again, isn't `values` already a `Ref`?
```suggestion
watch(values, () => {
```

**Review by Johann150 — APPROVED** [review]

**Comment by syuilo** [comment]
👍🏻

**Pull request merged**